Lib/test/test_cmd_line.py

```diff
@@ -161,6 +161,19 @@ def test_run_module_bug1764407(self):
         self.assertTrue(data.find(b'1 loop') != -1)
         self.assertTrue(data.find(b'__main__.Timer') != -1)
 
+    def test_relativedir_bug46421(self):
+        # Test `python -m unittest` with a relative directory beginning with ./
+        # Note: We have to switch to the project's top module's directory, as per
+        # the python unittest wiki. We will switch back when we are done.
+        defaultwd = os.getcwd()
+        projectlibpath = os.path.dirname(__file__).removesuffix("test")
+        os.chdir(projectlibpath)
```

Comment:
Take a look at `with change_cwd` from `os_helper`. It might be useful here.

Reply:
Thanks for the tip! That looks better.